admin documentation (#198)
maykinmedia/objects-api: docs/admin/object.rst

.. _admin_object:

=======
Objects
=======

> Creating and updating objects are usually done using the Objects API. :ref:`api_usage`

Reviewer: ```suggestion
Creating and updating objects is usually done using the Objects API. :ref:`api_usage`
```

describes how to do it in detail. But it's also possible to manage objects in the
admin interface. In this tutorial we will create and update objects using the Objects
admin.

Relate an object type
---------------------

Each object must belong to a particular object type created in the Objecttypes API.
The access to objects (the authorization) is also based on the related object types.
Therefore before creating the object we need to make the object type from the Objecttypes API
available in the admin interface of the Objects API. The section "Add an object type" of the
:ref:`admin_authorization` describes how to do it.

Create an object
----------------

Now we can create an object.  First, let's construct some tree data that matches our JSON schema
in the related object type:

.. code-block:: json

    {
        "boomgroep": "Solitaire boom",
        "boomhoogteactueel": 3,
        "leeftijd": 100,
        "meerstammig": false
    }

If you want, you can validate your JSON data against the JSON schema on `JSONschema.dev <https://jsonschema.dev>`_

> In the admin page of the Objects API click on the "add" button for the "Objects"
> resource.

Reviewer: ```suggestion
In the Objects admin dashboard, click on "Objects" and click on the "Add" button.
```

I like how you called the admin, the "Objects admin" in the introduction. I think we should keep this term throughout the document.
